.demo/env/charts/gatekeeper-operator.yaml

```diff
@@ -1,3 +1,22 @@
 charts:
     gatekeeper-operator:
         enabled: true
+        replicas: 1
+        disableValidatingWebhook: false
+        auditFromCache: false
```

Comment:
would we not want to keep an audit trail out of the box? Or is that not this flag?

Reply:
bump

Reply:
ok, sry for late reply,,, 

Enabling `auditFromCache` is useful if rules inside policies are querying more than just one Resource at a time. The most common example circulating is on "Ingress Listeners" -- if you want to create a rule that does not allow more than one `Gateway/Ingress` listening on the same hostname. (or any other examples of comparing multiple defined resources, just like the `--combine` flag in Conftest)

Reply:
Our defined policies are only checking some rules on the subjected resource. If we ever want to introduce comparing multiple resources... I guess it's a possibility, these rules will work just by onabling the `auditFromCache` property. Not sure about doing it in conftest, as it's an older version.

Reply:
```
Configuring Audit#

Audit violations per constraint: set --constraint-violations-limit=123 (defaults to 20)
Audit chunk size: set --audit-chunk-size=500 (defaults to 0 = infinite) to limit memory consumption of the auditing Pod
Audit interval: set --audit-interval=123 (defaults to every 60 seconds). Disable audit interval by setting --audit-interval=0
By default, the audit will request each resource from the Kubernetes API during each cycle of the audit. To instead rely on the OPA cache, use the flag --audit-from-cache=true. Note that this requires replication of Kubernetes resources into OPA before they can be evaluated against the enforced policies. Refer to the Replicating data section for more information.
```

https://open-policy-agent.github.io/gatekeeper/website/docs/audit/#configuring-audit